### ISSUE TITLE
entities cannot be setup anymore in 2022.3

### ISSUE THREAD

**Issue by kongo09** [kongo09/hass-dell-printer#1]
resulting error:

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 382, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 445, in _async_add_entity
    suggested_object_id = split_entity_id(entity.entity_id)[1]
  File "/usr/src/homeassistant/homeassistant/core.py", line 148, in split_entity_id
    raise ValueError(f"Invalid entity ID {entity_id}")
ValueError: Invalid entity ID .dell_printer_cyan
```

possibly related to:
https://github.com/home-assistant/core/pull/66835

**Comment by kongo09** [comment]
![image](https://user-images.githubusercontent.com/640121/156807512-d4868c7a-0228-44de-8a8d-11b09b6ba428.png)

**Comment by kongo09** [comment]
fixed by #2
